ui/admin/app/components/form/alias/index.hbs

```diff
@@ -138,12 +138,7 @@
     {{/if}}
   </Hds::Form::TextInput::Field>
 
-  <InfoField
-    @value={{@model.scopeModel.displayName}}
-    @icon='globe'
-    disabled={{form.disabled}}
-    as |F|
-  >
+  <InfoField @value={{@model.scopeModel.displayName}} @icon='globe' as |F|>
```

Comment:
Did we intend to remove `disabled={{form.disabled}}` and not replace with `readonly={{form.disabled}}`?

Reply:
Correct. We do not need to pass disabled or readonly to this component.